email.rst

```diff
@@ -158,16 +163,498 @@ file. For example, for `Amazon SES`_ (Simple Email Service):
+                <swiftmailer:delivery-address>dev@example.com</swiftmailer:delivery-address>
+            </swiftmailer:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/dev/swiftmailer.php
+        $container->loadFromExtension('swiftmailer', [
+            'delivery_addresses' => ["dev@example.com"],
+            'delivery_whitelist' => [
+                // all email addresses matching these regexes will be delivered
+                // like normal, as well as being sent to dev@example.com
+                '/@specialdomain\.com$/',
+                '/^admin@mydomain\.com$/',
+            ],
+        ]);
+
+In the above example all email messages will be redirected to ``dev@example.com``
+and messages sent to the ``admin@mydomain.com`` address or to any email address
+belonging to the domain ``specialdomain.com`` will also be delivered as normal.
+
+.. caution::
+
+    The ``delivery_whitelist`` option is ignored unless the ``delivery_addresses`` option is defined.
+
+Viewing from the Web Debug Toolbar
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can view any email sent during a single response when you are in the
+``dev`` environment using the web debug toolbar. The email icon in the toolbar
+will show how many emails were sent. If you click it, a report will open
+showing the details of the sent emails.
+
+If you're sending an email and then immediately redirecting to another page,
+the web debug toolbar will not display an email icon or a report on the next
+page.
+
+Instead, you can set the ``intercept_redirects`` option to ``true`` in the
+``dev`` environment, which will cause the redirect to stop and allow you to open
+the report with details of the sent emails.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/dev/web_profiler.yaml
+        web_profiler:
+            intercept_redirects: true
+
+    .. code-block:: xml
+
+        <!-- config/packages/dev/web_profiler.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:webprofiler="http://symfony.com/schema/dic/webprofiler"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/webprofiler
+                https://symfony.com/schema/dic/webprofiler/webprofiler-1.0.xsd">
+
+            <webprofiler:config
+                intercept-redirects="true"
+            />
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/dev/web_profiler.php
+        $container->loadFromExtension('web_profiler', [
+            'intercept_redirects' => 'true',
+        ]);
+
+.. tip::
+
+    Alternatively, you can open the profiler after the redirect and search
```

Comment:
what about adding this directly after the previous paragraph?
not as a tip, but as a two way options

personnaly i have never used intercept_redirects

Reply:
good note, but can you propose in a separate PR? This is just moved from a different part of the docs - i don't want to update anything in `email.rst` for this PR